lib/util/internalSerializables.js

```diff
@@ -169,7 +169,7 @@ module.exports = {
 		require("../InvalidDependenciesModuleWarning"),
 	Module: () => require("../Module"),
 	ModuleBuildError: () => require("../ModuleBuildError"),
-	ModuleDependencyWarning: () => require("../ModuleDependencyWarning"),
+	HarmonyLinkingError: () => require("../HarmonyLinkingError"),
```

Comment:
Dont quite understand this, so not sure if this is required

Reply:
Actually it's not. Neither `ModuleDependencyWarning` nor `HarmonyLinkingError` are serialized, so both could be omitted here. And you also need to omit the `makeSerializable` in the class.

Reply:
removed from this file. HarmonyLinkingError class did not previously have makeSerializable, so nothing to remove i think